roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -1,6 +1,12 @@
 filebeat.inputs:
-{{ filebeat_inputs | to_nice_yaml }}
-
+{% for item in filebeat_inputs %}
+- type: log
+  paths:
+    - {{ item['paths'] }}
```

Comment:
mm, won't this need to be a for loop? or else `paths: {{ item['paths'] | to_yaml }}`

Reply:
In case we have multiple paths ?

Reply:
yea, or else rename this to `path` so it's not plural

Reply:
Renamed it to path.